src/components/Laatija/laatija-utils.js

```diff
@@ -171,9 +171,10 @@ export const validate = {
   toteamispaivamaara: Validation.isPaivamaara
 };
 
+// Maa defaults to Finland as the transfer file does not have it as a field
 export const readRow = R.ifElse(
   R.compose(R.equals(R.length(dataFields)), R.length, R.split(';')),
-  R.compose(R.evolve(parse), R.zipObj(dataFields), R.split(';')),
+  R.compose(R.evolve(parse), R.assoc('maa', 'FI'), R.zipObj(dataFields), R.split(';')),
   R.always(null)
```

Comment:
This might get some refactoring in the near future.

Reply:
The structure of the code made it quite hard to put this anywhere else. I tried doing this in the parse function of R.evolve, but seems that evolve does not call functions for fields that are not in the parse spec :/

If you have any ideas, feel free to suggest before I merge